csrc/executor_utils.cpp

```diff
@@ -912,8 +926,10 @@ ExpressionEvaluator bindInputs(
     bool check_consistency) {
   FUSER_PERF_SCOPE("executor_utils::bindInputs");
 
+  // args may contains more than just inputs, but inputs are always at the
+  // beginning.
   TORCH_INTERNAL_ASSERT(
-      kernel->inputs().size() == args.size(),
+      kernel->inputs().size() <= args.size(),
```

Comment:
nitpick. maybe update the assert message.

Reply:
updated